internal/cmd/oohelperd/internal/nwcth/dns.go

```diff
@@ -0,0 +1,39 @@
+package nwcth
+
+import (
+	"context"
+	"net"
+
+	"github.com/apex/log"
+	"github.com/ooni/probe-cli/v3/internal/engine/experiment/nwebconnectivity"
+	"github.com/ooni/probe-cli/v3/internal/engine/netx"
```

Comment:
*Dependency question*: I see there is a bunch of places where we still depend on `netx`. Is this dependency functionally required or could it be removed by using `netxlite` + `errorsx`?

Reply:
Added comment so we remember about this.